<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintLogDlg</class>
 <widget class="QDialog" name="PrintLogDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打印数据仓库</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>打印日期(YYYY-MM-DD)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtDate">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>9999-99-99</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>凭证名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtVoucherName"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuery">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClean">
         <property name="text">
          <string>清理日志</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tvLogDetails">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="btnFirst">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnPrev">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnNext">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnLast">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>末页</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>PrintLogDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
